build/husky.sh

```diff
@@ -0,0 +1,4 @@
+echo "Husky installing ..."
+npm install husky --save-dev
+npx husky install
+echo "Husky is already installed and hooks enabled."
```

Comment:
The wording here seems off. Every time this script is executed, the following text will be displayed:

```bash
Husky installing ...
Husky is already installed and hooks enabled.
```

Consider removing "is already" from the last sentence since no checks are actually being done to determine if husky was previously installed:

```bash
Husky installing ...
Husky installed and hooks enabled.
```